static/main.css

```diff
@@ -52,9 +52,6 @@ input:focus {
 .user-feedback.success {
   background-color: #4caf50;
 }
-img {
-  max-width: 500px;
```

Comment:
I think this made sure that the images are somehow more or less shown in their full size. But not 100% sure.